movie/models.py

```diff
@@ -0,0 +1,125 @@
+    korean_title       = models.CharField(max_length=100)
+    english_title      = models.CharField(max_length=100, null=True)
+    release_date       = models.DateField(auto_now=False, auto_now_add=False)
+    nationality        = models.CharField(max_length=45)
+    number_of_audience = models.PositiveIntegerField()
+    description        = models.TextField()
+    running_time       = models.TimeField()
+    is_watcha          = models.BooleanField(default=False)
+    category           = models.ForeignKey(Category, on_delete=models.CASCADE)
+    main_sort          = models.ForeignKey('MainSort', null=True, on_delete=models.SET_NULL)
+    movie_like         = models.ManyToManyField(User, through='MovieLike', related_name='movie_movie_like')
+    user_star          = models.ManyToManyField(User, through='UserStar', related_name='movie_user_star')
+    comment            = models.ManyToManyField(User, through='Comment', related_name='movie_comment')
+    collection         = models.ManyToManyField(User, through='Collection', related_name='movie_collection')
+    genre              = models.ManyToManyField('Genre', through='MovieGenre', related_name='movie_genre')
+    
+    class Meta:
+        db_table = 'movies'
+
+
+class MainSort(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'main_sorts'
+
+
+class MovieImage(models.Model):
+    image_url = models.TextField()
+    movie     = models.ForeignKey(Movie, on_delete=models.CASCADE)
+    
+    class Meta:
+        db_table = 'movie_images'
+
+
+class Gallery(models.Model):
+    image_url = models.TextField(null=True)
+    video_url = models.TextField(null=True)
+    movie     = models.ForeignKey(Movie, on_delete=models.CASCADE)
+    
+    class Meta:
+        db_table = 'gallery'
+
+
+class MovieLike(models.Model):
+    user  = models.ForeignKey(User, on_delete=models.CASCADE)
+    movie = models.ForeignKey(Movie, on_delete=models.CASCADE)
+    
+    class Meta:
+        db_table = 'movie_likes'
+
+
+class UserStar(models.Model):
+    user  = models.ForeignKey(User, on_delete=models.CASCADE)
+    movie = models.ForeignKey(Movie, on_delete=models.CASCADE)
+    star  = models.IntegerField()
+
+    class Meta:
+        db_table = 'user_stars'
+
+
+class Panel(models.Model):
+    name      = models.CharField(max_length=45)
+    movie     = models.ForeignKey(Movie, on_delete=models.CASCADE)
+    role      = models.CharField(max_length=100)
+    character = models.CharField(max_length=45, null=True)
+    image_url = models.TextField(null=True)
+   
+    class Meta:
+        db_table = 'panels'
+
+
+class Comment(models.Model):
+    user         = models.ForeignKey(User, on_delete=models.CASCADE)
+    movie        = models.ForeignKey(Movie, on_delete=models.CASCADE, related_name='comment_movie')
+    content      = models.TextField()
+    comment_like = models.ManyToManyField('user.User', through='CommentLike', related_name='comment_like')
```

Comment:
user.User 에서 앞의 `user` 와 `.User` 가 무엇을 의미하는지 설명부탁드립니다!

반복되는 comment 는 제거해도 됩니다.
```suggestion
    like = models.ManyToManyField('user.User', through='CommentLike', related_name='comment_like')
```

Reply:
user는 앱, User는 user앱 내부의 User 클래스입니다. User로 import했기 때문에 그냥 User로 작성해도 괜찮을 것 같네요!